main.go

```diff
@@ -22,6 +22,8 @@ import (
 	"os"
 	"time"
 
+	"volcano.sh/resource-exporter/pkg/machineinfo"
```

Comment:
Pls keep the imported packages in order. In general, go package first, third package such as k8s second, packages in volcano last.

Reply:
Is there any sort rule file or tool? Or we need to adjust it manually?

Reply:
> Is there any sort rule file or tool? Or we need to adjust it manually?

It's by manually temporarily.